src/gstools/variogram/estimator.pyx

```diff
@@ -342,7 +345,7 @@ def structured(
 
 def ma_structured(
     const double[:, :] f,
-    const bint[:, :] mask,
+    uint8[:, :] mask,  # numpy's bools are 8bit vars
```

Comment:
But we passed the masks as np.int32 before. Was that not working anymore?

Reply:
I have absolutely no clue! But it seems, if we want to use numpy's bool types, `unit8` is the right type to use in Cython. The only thing that I changed that breaks the old code is that we now use function pointers in Python. But don't ask me how that could explain anything ;-)

Reply:
I guess the problem is your if condition for converting the array: `not config.USE_GSTOOLS_CORE and config._GSTOOLS_CORE_AVAIL`

This should be only `not config.USE_GSTOOLS_CORE` since if GSTools-Core is not available (and we obviously want to use cython) this will not evaluate to true and the conversion for cython is not done.